neovim/plugin/script_host.py

```diff
@@ -63,6 +63,12 @@ def setup(self, nvim):
         self.saved_stderr = sys.stderr
         sys.stdout = RedirectStream(lambda data: nvim.out_write(data))
         sys.stderr = RedirectStream(lambda data: nvim.err_write(data))
+        # Setup the DirChanged listener
```

Comment:
Hmm maybe we can use the host autocmd infrastructure here. (We need to add a flag to not start the host for the autocmd, but that would be useable in general)

Reply:
Actually that would require a bit bigger refactor than I thought, so let's keep this as is for now (and maybe revisit at `UpdateRemotePlugins` refactor)